src/layer/AbstractSentinelHubV1OrV2Layer.ts

```diff
@@ -105,4 +105,31 @@ export class AbstractSentinelHubV1OrV2Layer extends AbstractLayer {
       hasMore: response.data.hasMore,
     };
   }
+
+  protected getFindDatesAdditionalParameters(): Record<string, any> {
+    return {};
+  }
+
+  public async findDates(bbox: BBox, fromTime: Moment, toTime: Moment): Promise<Moment[]> {
+    if (!this.dataset.findDatesUrl) {
+      throw new Error('This dataset does not support searching for dates');
+    }
+
+    const payload = bbox.toGeoJSON();
+    const params = {
+      timefrom: fromTime.toISOString(),
+      timeto: toTime.toISOString(),
+      ...this.getFindDatesAdditionalParameters(),
+    };
+
+    const url = `${this.dataset.findDatesUrl}?${stringify(params, { sort: false })}`;
+    const response = await axios.post(url, payload, {
+      headers: {
+        'Content-Type': 'application/json',
+        'Accept-CRS': 'EPSG:4326',
+      },
+    });
+
+    return response.data.map((d: string) => moment.utc(d));
```

Comment:
In another PR (#41) we are trying to get rid of moment.js objects in the library interface, so it would probably make sense to change this here also.I would suggest something along the lines of:

```
return response.data.map((d: string) => moment.utc(d).toDate());
```

edit: ...in other places too...

Reply:
Thanks, changed. I also changed that the parameters (`fromTime`, `toTime`) of `findDates()` are `Date` objects.